editor.planx.uk/src/pages/FlowEditor/components/Team/components/lib/optimisticallyUpdateTable.tsx

```diff
@@ -0,0 +1,18 @@
+import { getRandomNumberInRange } from "@planx/components/shared/utils";
+import React from "react";
+
+import { AddNewEditorFormValues, TeamMember } from "../../types";
+
+export const optimisticallyUpdateTable = (
+  values: AddNewEditorFormValues,
+  setTableData: React.Dispatch<React.SetStateAction<TeamMember[]>>,
+  tableData: TeamMember[],
+) => {
+  const randomNumber = getRandomNumberInRange(100, 200); // each table row needs a unique numerical key so randomise for now
```

Comment:
Seems a bit hacky, and also not scalable if a council has more than 100 users in this team Editors table... Would welcome thoughts on improving!